### PR TITLE
Bluetooth: Mesh: Move netcore specific configuration to samples

### PR DESCRIPTION
**Pull request by PavelVPV**
This configuration is specific for the samples, thus network core should be reconfigured there.

### CODE REVIEW

**Review by ludvigsj — APPROVED** [review]

File: samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copy controller configuration from prj.conf
+CONFIG_BT_BUF_ACL_RX_SIZE=73
```
Comment (alxelax) [inline samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf:2]:
```suggestion
CONFIG_BT_BUF_ACL_RX_SIZE=37
```
here and below

Comment (PavelVPV) [reply samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf:2]: No, check with nrf52840. Default for mesh is 73 if GATT is enabled

Comment (PavelVPV) [reply samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf:2]: https://github.com/zephyrproject-rtos/zephyr/blob/8959b838bf15118dadc12c21e3d4ca5a92039c77/subsys/bluetooth/common/Kconfig#L48-L53

**Comment by PavelVPV** [comment]
CI running samples: https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Ftest-fw-nrfconnect-ble_mesh/detail/PR-4025/2/

**Comment by PavelVPV** [comment]
Changelog is not required as the bug was added during release testing phase.

File: samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf (hunk 1 above)
Comment (alxelax) [reply samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf:2]: It is a bit expensive to keep 73 bytes just to not fragment public key exchange that happens very rarely (once per provisioning procedure). It is not blocker, but I'd recommend revising this digit. Nothing critical if public key exchange will be fragmented.

Comment (PavelVPV) [reply samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf:2]: It is as it is. Proper fix will be done as part of NCSDK-21755

**Review by alxelax — APPROVED** [review]

File: samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf (hunk 1 above)
Comment (PavelVPV) [reply samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf:2]: Currently, the values are aligned with what the default to if sample is build for nRF52840.

**Pull request merged**